web/sites/default/config/webform.webform.report_campsite.yml

```diff
@@ -368,7 +375,7 @@ settings:
   form_open_message: ''
   form_close_message: ''
   form_exception_message: ''
-  form_previous_submissions: false
+  form_previous_submissions: true
```

Comment:
Enables a "you're previously submitted this form" message on the Drupal webform. It is only visible to logged in users, so the impact to community members is nil. It's been re-enabled because that's the default for webforms (not sure why it was turned off).